src/dataflow-types/src/types.rs

```diff
@@ -899,6 +900,18 @@ pub mod sources {
         pub mode: DebeziumMode,
     }
 
+    #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+    pub struct DebeziumTransactionMetadata {
+        pub tx_metadata_global_id: GlobalId,
```

Comment:
NIT: I would drop the `tx_` prefix since it is implied by the type name and will make them shorter